extensions/nsis-reference/src/index.tsx

```diff
@@ -1,6 +1,8 @@
-import { ActionPanel, Detail, Icon, List, PushAction } from "@raycast/api";
+import { ActionPanel, Detail, Icon, List, Action } from "@raycast/api";
+// @ts-expect-error Cannot find module '@nsis/docs' or its corresponding type declarations.ts (2307)
 import { Callbacks, Commands, Includes, Plugins, Variables } from "@nsis/docs";
```

Comment:
I will fix this later today in `@nsis/docs`, so we can get rid off this comment

Reply:
That was super fast of you!
Unfortunately, now I'm getting a new error:
`Cannot find module '@nsis/docs' or its corresponding type declarations.
  There are types at '[PATH]/raycast-extensions/extensions/nsis-reference/node_modules/@nsis/docs/dist/index.d.ts', but this result could not be resolved under your current 'moduleResolution' setting. Consider updating to 'node16', 'nodenext', or 'bundler'.ts (2307)`.
  
These settings are the Raycast set defaults but I can make the error go away by adding the path to tsconfig.json like so:

```json{
  "$schema": "https://json.schemastore.org/tsconfig",
  "include": ["src/**/*", "raycast-env.d.ts"],
  "compilerOptions": {
    ....
    "paths": {
      "@nsis/docs": [
        "./node_modules/@nsis/docs/dist/index.d.ts"
      ]
    }
  }
}`

Or we can let the previous comment remain. What would you like to do?